apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/plan-step/styles.scss

```diff
@@ -10,5 +11,10 @@
 	.plans-grid__details-container {
 		position: relative;
 		@include onboarding-block-edge2edge-container;
+
+		@media ( min-width: $plans-grid-max-page-width ) {
+			margin-left: 0;
+			margin-right: 0;
+		}
```

Comment:
This change lies in Editing toolkit because it only applies in Step by Step launch (not gutenbording).